### PR TITLE
f-searchbox@v4.0.0 beta.13

### PR DESCRIPTION
**Pull request by kevinrodrigues**
Fixed NO & DK from not submitting correctly as the attributes on the component were wrong.` Postcode > Where`

Key Attribute required: 

Screen Shot 2021-01-05 at 1 53 21 PM

Was:

Screen Shot 2021-01-05 at 1 54 06 PM

### CODE REVIEW

**Review by simonsandersje — DISMISSED** [review]

File: packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue [hunk 1]
```diff
@@ -50,6 +50,20 @@ export default {
...
         FormSearchInnerFieldWrapper
     },
+
+    directives: {
+        /**
+         * Extract object attributes and set them on the input component.
+         *
+         * */
+        setAttributes (element, directive) {
+            for (let i = 0, keys = Object.keys(directive.value); i < keys.length; i++) {
```
Comment (minrwhite) [inline packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue]: Wouldn't [`Object.entries()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries)`.forEach()` work here?

Comment (kevinrodrigues) [reply packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue]: It would. Ill add that 👍

Comment (kevinrodrigues) [reply packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue]: Fixed.

Comment (kevinrodrigues) [reply packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue]: Double checked storybook by running build and serve locally and works fine with this change.

**Review by simonsandersje — APPROVED** [review]

**Review by matthewhardern — APPROVED** [review]

**Pull request merged**